src/main/java/service/dao/CommentDAO.java

```diff
@@ -70,6 +73,14 @@ public Comment getComment(int commentId) {
         }
     }
 
+    /**
+     * @return all comments
+     */
+    @Override
+    public ArrayList<Comment> getAllComments() {
+        return (ArrayList<Comment>) jdbcTemplate.query(QUERY_ALL, new CommentDaoMapper());
+    }
```

Comment:
This is a bit unnecessary, there is no situation where we will need to get all comments. The comments will belong to different parent reviews.

Reply:
I need this data access call because in my response model I need to return the new comments list